.github/workflows/base-ci-goreleaser.yaml

```diff
@@ -67,7 +67,7 @@ jobs:
         uses: goreleaser/goreleaser-action@286f3b13b1b49da4ac219696163fb8c1c93e1200 # v6.0.0
         with:
           distribution: goreleaser-pro
-          version: latest
+          version: v2.2.0
```

Comment:
Could you file a separate PR for this?

Reply:
I filed it as #667. Should I remove the change from this PR?

Reply:
It should go away after I merge #667 and you merge main into this PR